awx/ui_next/src/screens/Login/Login.jsx

```diff
@@ -61,13 +61,16 @@ function AWXLogin({ alt, i18n, isAuthenticated }) {
       const logoSrc = custom_logo
         ? `data:image/jpeg;${custom_logo}`
         : loginLogoSrc;
+      if (login_redirect_override && !isAuthenticated(document.cookie)) {
+        return window.location.replace(login_redirect_override);
```

Comment:
@AlexSCorey 

Burying this deep within an async callback (and conditionally changing it's return structure) makes me a little uneasy.

We have similar handling for conditional redirects further down in the component definition that use the typical `react-router` semantics and, if possible, I think we should do something similar.

If I'm understanding this correctly, I think all you'd need to do is add:

```jsx
  if (login_redirect_override) {
    return <Redirect to={login_redirect_override} />;
  }
```

to [this line](https://github.com/ansible/awx/blob/devel/awx/ui_next/src/screens/Login/Login.jsx#L115)

Reply:
or, if react-router doesn't like the external url:

```jsx
  if (login_redirect_override) {
    window.location.replace(login_redirect_override);;
  }

Reply:
React router does not like redirecting outside of the app.  Using option 1